tests/eventEmitterOff.test.js

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable max-len */
+
+'use strict';
+
+const {
+  EventEmitter,
+} = require('../src/MyEventEmitter');
+
+describe('EventEmitter "off" method tests', () => {
+  let emitter;
+
+  beforeEach(() => {
+    emitter = new EventEmitter();
+  });
+
+  test('should remove the specified listener from the listener array', () => {
+    const listener = jest.fn();
+
+    emitter.on('event', listener);
+    emitter.emit('event');
+    expect(listener).toHaveBeenCalledTimes(1);
+
+    emitter.off('event', listener);
+    emitter.emit('event');
+    expect(listener).toHaveBeenCalledTimes(1);
+  });
+
+  test('should remove a listener added multiple times, one at a time', () => {
+    const listener = jest.fn();
+
+    emitter.on('event', listener);
+    emitter.on('event', listener);
```

Comment:
are you sure that the same listener can be added twice?
please check if Node.js works this way